src/Orders/OrderStatus.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace ResellerClub\Orders;
+
+class OrderStatus
+{
+    /**
+     * @var string
+     */
+    private $status;
+
+    /**
+     * Create a new order status instance.
+     *
+     * @param string $status
+     *
+     * @return self
+     */
+    public function __construct(string $status)
+    {
+        $this->status = strtolower($status);
+    }
+
+    /**
+     * Get the string representation of the object.
+     *
+     * @return string
+     */
+    public function __toString()
+    {
+        return $this->status;
+    }
+
+    /**
+     * Is the order marked as "Active".
+     *
+     * @return bool
+     */
+    public function isActive(): bool
+    {
+        return (string) $this->status === 'active';
+    }
+
+    /**
+     * Is the order marked as "InActive".
+     *
+     * @return bool
+     */
+    public function isInactive(): bool
+    {
+        return (string) $this->status === 'inactive';
```

Comment:
Does this string need normalising (i.e. making all lowercase) for comparison? As in, could Reseller Club return a status string like “inActive”?

Reply:
Done that in the __construct

Reply:
@OAFCROB Ah, cool. Missed that. Sorry!